### PR TITLE
LRQA-78394

### PR DESCRIPTION
**Pull request by kyle-miho**
https://issues.liferay.com/browse/LRQA-73894

### CODE REVIEW

**Comment by kyle-miho** [comment]
Hi @rafaprax This review is mainly to confirm the formatting of the newly added entity to the service.xml
cc: @vicnate5 @tinatian 
https://github.com/liferay-core-infra/liferay-portal/pull/1442/commits/fde88ed3bd5751236c6d3fa68b81780e5aa60c27

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:sf

File: modules/util/portal-tools-service-builder-test-service/service.xml [hunk 1]
```diff
@@ -353,6 +353,23 @@
...
+		<column name="groupId" type="long" />
+
+		<!-- Other fields -->
+		<column name="columnToRename" type="String" />
+
+        <!-- Finders -->
+        <finder name="ColumnToRename" return-type="RenameFinderColumnEntry">
+            <finder-column name="columnToRename"/>
+        </finder>
+	</entity>
```
Comment (rafaprax) [inline modules/util/portal-tools-service-builder-test-service/service.xml:372]: Hey @kyle-miho, please fix the indentation of the <finder> section.

**Comment by kyle-miho** [comment]
ci:reopen

**Pull request closed without merging**